rust/template/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(non_snake_case, non_camel_case_types, non_upper_case_globals, unused_parens, non_shorthand_field_patterns, dead_code, overflowing_literals)]
+#![allow(unused_imports, non_snake_case, non_camel_case_types, non_upper_case_globals, unused_parens, non_shorthand_field_patterns, dead_code, overflowing_literals, clippy::ptr_arg)]
```

Comment:
How come you didn't have to add clippy in Cargo.toml for this to work?

Reply:
I think this was a compiler warning, not from clippy.

Reply:
I am wondering about `clippy::ptr_arg`.

Reply:
Hmm maybe because I installed clippy globally? Does it not work for you?

Reply:
It does, actually. I guess clippy is not sufficiently integrated with rustc?  Or perhaps rust just knows to ignore these annotations.

Reply:
Yes, turns out rust ignores it if clippy isn't installed: https://doc.rust-lang.org/reference/attributes/diagnostics.html#tool-lint-attributes